README.md

```diff
@@ -33,111 +33,26 @@ For details see the reference documentation on Hackage.
 Each module lines up with the hierarchy of
 [documentation from the Github API](http://developer.github.com/v3/).
 
-Each function has a sample written for it.
+Request functions (ending with `R`) construct a data type with can be executed
+in `IO` by `executeRequest` functions. They are all listed in `Github.All` module.
 
-All functions produce an `IO (Either Error a)`, where `a` is the actual thing
+IO functions produce an `IO (Either Error a)`, where `a` is the actual thing
 you want. You must call the function using IO goodness, then dispatch on the
 possible error message. Here's an example from the samples:
 
-    import qualified Github.Users.Followers as Github
-    import Data.List (intercalate)
+Many function has a sample written for it.
```

Comment:
The word cases don't match, but I couldn't come up with a clear way to say this. How about: "Many functions have samples under the `samples/` directory."